### ISSUE TITLE
Cannot see tracks that are in iCloud Music Library

### ISSUE THREAD

**Issue by richtong** [tattwamasi/TeslaTunes#18]
Does this utility work with the iCloud music library. My songs and playlists are stored up there and it keeps saying. The track "XXXX" has no location specified. The track is downloaded and does play by the way.

**Comment by richtong** [comment]
Looking at another post, this looks like a feature, not a bug :-) Maybe good to document in the README.md, I can do it and make a Pull Request if you like...

**Comment by lraesly** [comment]
Apple’s subscription music library won’t sync with anything as they are copy protected songs.  If you’re referring to the traditional iCloud syncing of purchased or personally ripped songs, you need to download them first and then they’ll sync.  TeslaTunes can’t sync them if they’re not stored locally.

Hope this helps..

Lee

**Comment by richtong** [comment]
Makes sense, the error through me, might want to add the readme. Awesome tool!

Thanks
Rich

—
The only way of discovering the limits of the possible is to venture a little way past them into the impossible. Arthur C. Clarke. https://en.wikipedia.org/wiki/Clarke's_three_laws <https://en.wikipedia.org/wiki/Clarke's_three_laws>
—
https://tongfamily.com <https://tongfamily.com/> | https://medium.com/@richt <https://medium.com/@richt> | https://instagram.com/richtong <https://instagram.com/richtong> | https://github.com/richtong <https://github.com/richtong> | https://linkedin.com/in/richtong1 <https://linkedin.com/in/richtong1> | https://fb.me/richtong1 <https://fb.me/richtong1> | @richtong <https://twitter.com/richtong> | +1-206-963-9029

**Comment by tattwamasi** [comment]
Will add a note next time I touch the readme.
